packages/docusaurus-theme-classic/src/theme/CodeBlock/index.tsx

```diff
@@ -124,8 +124,7 @@ export default ({
   if (metastring && codeBlockTitleRegex.test(metastring)) {
     // Tested above
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    codeBlockTitle = metastring
-      .match(codeBlockTitleRegex)![1];
+    codeBlockTitle = metastring.match(codeBlockTitleRegex)![1];
```

Comment:
we already have this code formatting fixed on master

Reply:
I just need to rebase, will do that when I make the other required changes. Sorry for the confusion